code/obj/flock/structure/relay.dm

```diff
@@ -87,6 +93,14 @@
 		M.playsound_local(M, "sound/ambience/spooky/Flock_Reactor.ogg", 35, 0, 2)
 		boutput(M, "<span class='flocksay bold'>You hear something unworldly coming from the <i>[dir2text(get_dir(M, center_loc))]</i>!</span>")
 
+/obj/flock_structure/relay/proc/convert_turfs()
+	var/list/turfs = circular_range(get_turf(src), src.conversion_radius)
+	SPAWN(0)
+		for (var/turf/T as anything in turfs)
+			if (istype(T, /turf/simulated) && !isfeathertile(T))
+				sleep(0.01 SECONDS)
```

Comment:
big unfan, please use a lagcheck or a manual counter or something

Reply:
Just decided to remove the effect, wasn't able to get desired one

Reply:
why not it's literally a one line change to use a lagcheck

Reply:
I did try using a lagcheck, still seemed to be much faster than the effect I was going for

Reply:
I tried doing it again, I guess the effect just ends up looking a bit different closer to the relay, can see it a little more clearly further away from the relay